src/main/java/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -321,10 +324,11 @@ public void onConfigurationChanged(Configuration newConfig) {
     composeText.setTransport(sendButton.getSelectedTransport());
     quickAttachmentDrawer.onConfigurationChanged();
 
-    if (emojiDrawerStub.resolved() && container.getCurrentInput() == emojiDrawerStub.get()) {
+    if (emojiPicker != null && container.getCurrentInput() == emojiPicker) {
       container.hideAttachedInput(true);
     }
 
+    reloadEmojiPicker();
```

Comment:
Could just set emojiPicker to null instead of fully reloading it on orientation change, because it's hidden anyway